assets/js/phoenix_live_view.js

```diff
@@ -2000,12 +2022,19 @@ export class View {
     return(
       this.liveSocket.wrapPush(() => {
         return this.channel.push(event, payload, PUSH_TIMEOUT).receive("ok", resp => {
-          if(resp.diff || ref !== null){ this.update(resp.diff || {}, payload.cid, ref) }
+          let hookReply = null
+          if(ref !== null){ DOM.undoRefs(ref, this.el) }
+          if(resp.diff){
+            this.applyDiff("update", resp.diff, ({diff, reply, events}) => {
+              hookReply = reply
```

Comment:
What if `applyDiff` returns the reply instead of passing it to the anonymous function? Would that be any better?